yarn.lock

```diff
@@ -3,1935 +3,2404 @@
 
 
 "@apidevtools/json-schema-ref-parser@^9.0.6":
```

Comment:
Are dependency upgrades in the scope of this task? applies to all changes in file.

Reply:
Not necessarily. I included this just because it was automatically changed. Perhaps can be left out if you think it is better?

Reply:
Yes, let's not touch dependencies if it's not part of the scope.

Reply:
done